wsd/AdminModel.cpp

```diff
@@ -895,6 +897,24 @@ int AdminModel::getPidsFromProcName(const std::regex& procNameRegEx, std::vector
     return pidCount;
 }
 
+int AdminModel::getAssignedKitPids(std::vector<int> *pids)
+{
+    return getPidsFromProcName(std::regex("kitbroker_.*"), pids);
+}
+
+int AdminModel::getUnassignedKitPids(std::vector<int> *pids)
+{
+    return getPidsFromProcName(std::regex("kit_spare_.*"), pids);
+}
+
+int AdminModel::getKitPidsFromSystem(std::vector<int> *pids)
+{
+    int count = getAssignedKitPids(pids);
+    count += getUnassignedKitPids(pids);
+
+    return count;
```

Comment:
`return getAssignedKitPids(pids) + getUnassignedKitPids(pids);` ?

Reply:
Well, I prefer to do like this just for debug purpose. Just to be able to put a breakpoint at second call.